src/myrtlespeech/protos/train_config.proto

```diff
@@ -29,10 +30,12 @@ message TrainConfig {
 
   Dataset dataset = 9;
 
-  oneof supported_shuffles {
+  oneof shuffle_strategy {
+    // Mantain a sequential batch order.
+    SequentialBatches sequential_batches = 10;
     // Shuffle batches before every epoch.
-    bool shuffle_batches_before_every_epoch = 10;
+    RandomBatches random_batches = 11;
+    // Sequential for the first epoch and random for the following ones.
+    SortaGrad sortagrad = 12;
```

Comment:
Nit: messages defined using CamelCase have variable names written using snake_case. This convention makes this line `SortaGrad sorta_grad`.

Reply:
Changed it!
I actually thought Sortagrad was a single name instead of two, that's why I didn't use any underscore